src/room/GroupCallInspector.tsx

```diff
@@ -354,30 +415,30 @@ function useGroupCallState(client, groupCall, pollCallStats) {
     //   dispatch({ type: "call_hangup", call });
     // }
 
-    function onReceivedVoipEvent(event) {
-      dispatch({ type: "received_voip_event", event });
+    function onReceivedVoipEvent(event: MatrixEvent) {
+      dispatch({ type: ClientEvent.ReceivedVoipEvent, event });
     }
 
-    function onSendVoipEvent(event) {
-      dispatch({ type: "send_voip_event", event });
+    function onSendVoipEvent(event: MatrixEvent) {
+      dispatch({ type: CallEvent.SendVoipEvent, event });
     }
-
-    client.on("RoomState.events", onUpdateRoomState);
+    const firstCall = groupCall.calls[0];
+    client.on(RoomStateEvent.Events, onUpdateRoomState);
     //groupCall.on("calls_changed", onCallsChanged);
-    groupCall.on("send_voip_event", onSendVoipEvent);
+    firstCall.on(CallEvent.SendVoipEvent, onSendVoipEvent);
```

Comment:
hmm, I suspect it probably was not what was intended, although it must have been getting the events somehow - probably worth finding out why. It should get events for all the calls rather than just the first.

Reply:
This was indeed the case. There is a reemitter on GroupCall. The events send by the reemitter were not part of the GroupCallType. This is fixed with: https://github.com/matrix-org/matrix-js-sdk/pull/2527